src/test/java/com/radixdlt/identifiers/RadixAddressTest.java

```diff
@@ -77,7 +78,7 @@ public void createAddressFromPublicKey() throws CryptoException {
 	@Test(expected = DecoderException.class)
 	public void createAddressFromBadPublicKey() throws CryptoException, DecoderException {
 		ECPublicKey publicKey = new ECPublicKey(Base64.decode("BADKEY"));
-		new RadixAddress(magicByte(), publicKey);
+		assertNotNull(new RadixAddress(magicByte(), publicKey));
```

Comment:
same thing as my other comment... this should not be a RadixAddress test, but an ECPublicKey test, since the RadixAddress constructor is never called.

Reply:
Right, fixed as part of the changes to use `assertThatThrownBy(...)`